titus_isolate/isolate/workload_manager.py

```diff
@@ -48,18 +61,30 @@ def __add_workload(self, workload):
         workload_id = workload.get_id()
         self.__workloads[workload_id] = workload
 
+        allocator = self.__cpu_allocator
         if workload.get_type() == STATIC:
             current_cpu = copy.deepcopy(self.get_cpu())
-            self.__cpu_allocator.assign_threads(workload)
-            updates = get_updates(current_cpu, self.__cpu_allocator.get_cpu())
+            try:
```

Comment:
Might be slightly neater if this `try/catch` were factored out into something like

```python
def ___allocate(workload):
    ...
    return allocator.get_cpu()
```

Reply:
Done, thanks